makefile.shade

```diff
@@ -360,7 +360,7 @@ var buildTarget = "compile"
 
                 if (IsMono)
                 {
-                    Exec("build.sh", buildTarget, repo);
+                    Exec(Path.Combine(repo, "build.sh"), buildTarget, repo);
```

Comment:
Minor but worthwhile for consistency: Do the same fixup w/ `build.cmd` in the `else` branch here and below.